src/parsers.rs

```diff
@@ -202,21 +204,44 @@ impl StructuredLogParser for InductorOutputCodeParser {
                 .as_ref()
                 .and_then(|p| Path::file_stem(p))
                 .map_or_else(
-                    || PathBuf::from("inductor_output_code.txt"),
+                    || PathBuf::from("inductor_output_code.html"),
                     |stem| {
                         let mut r = OsString::from("inductor_output_code_");
                         r.push(stem);
-                        r.push(OsStr::new(".txt"));
+                        r.push(OsStr::new(".html"));
```

Comment:
It'd be nice to still dump the txt file, in case someone wants to directly run it, but I guess it's not a big deal, certainly only one file saves us upload time.